pkg/testutil/store.go

```diff
@@ -0,0 +1,125 @@
+package testutil
+
+import (
+	"context"
+	"errors"
+	"io"
+	"testing"
+
+	"github.com/improbable-eng/thanos/pkg/store/storepb"
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/tsdb/chunkenc"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// WarningSeries is a series that triggers partial error warning.
+var WarningSeries = storepb.Series{Labels: []storepb.Label{{Name: "WarningSeries"}}}
```

Comment:
Defining such sentinel values in another package (especially as a variable) itches me a bit. Especially because it sets some odd-ish values we do not see immediately in proximity to the rest of the test definition.
What are we saving as opposed to just specifying it inline in tests where we need it.

Reply:
Yea, this one is ugly. Well I wanted to mix warnings with series, but I can combine this together differently (e.g in oneof style)